engine/profile_tactic.mli

```diff
@@ -47,8 +47,10 @@ val do_profile_gen :
 
 val set_profiling : bool -> unit
 
+val get_profiling : unit -> bool
+
 (* Cut off results < than specified cutoff *)
-val print_results : cutoff:float -> unit
+val print_results : cutoff:float option -> unit
```

Comment:
Should this get a note in the OCaml interface changelog?

Reply:
The ocaml changelog is only for things which are tricky to port, and even then we often don't update it